content/en/docs/instrumentation/swift/instrumentation.md

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
Consider naming the file `automatic.md` like for other language SIGs. (WDYT @cartermp et al.?)